alpaka/plugin/src/main/java/ch/ubique/gradle/alpaka/task/LauncherIconLabelTask.kt

```diff
@@ -62,23 +57,14 @@ abstract class LauncherIconLabelTask : DefaultTask() {
 		}
 		val generatedIconDir = generatedIconDir.get().asFile
 
-		val gradleLastModified = listOf(
-			File(project.projectDir, "build.gradle").lastModified(),
-			File(project.projectDir, "build.gradle.kts").lastModified(),
-			File(project.rootDir, "build.gradle").lastModified(),
-			File(project.rootDir, "build.gradle.kts").lastModified(),
-			File(project.rootDir, "settings.gradle").lastModified(),
-			File(project.rootDir, "settings.gradle.kts").lastModified(),
-			File(project.rootDir, "gradle/libs.versions.toml").lastModified(),
-		).max()
-
-		val flavorNames = setOf(fullFlavorName) + partialFlavorNames
-		val resDirs = project.getResDirs(flavorNames)
+		val gradleLastModified = buildLogicFiles.files
```

Comment:
val gradleLastModified = buildLogicFiles.files.maxOfOrNull { it.lastModified() } ?: 0L